examples/embed/embed_themed.py

```diff
@@ -0,0 +1,66 @@
+import io
+
+from jinja2 import Template
+
+from bokeh.embed import components
+from bokeh.resources import INLINE
+from bokeh.util.browser import view
+from bokeh.io import curdoc, show
+from bokeh.themes import Theme
+from bokeh.plotting import figure
+
+x1 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+y1 = [0, 8, 2, 4, 6, 9, 5, 6, 25, 28, 4]
+
+p1 = figure(title='PINK TITLE')
+p1.background_fill_color = "#999999"
+p1.border_fill_color = "#999999"
```

Comment:
these colors can go in the theme, sorry I was not explicit but I was suggesting making the theme have a bit more content.